modules/sbo-data-model.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-data-model_{context}"]
+= Data model
+
+Service binding annotations follow the convention:
+
+[source,yaml]
+----
+service.binding(/<NAME>)?: "<VALUE>|(path=<JSONPATH_TEMPLATE>(,objectType=<OBJECT_TYPE>)?(,elementType=<ELEMENT_TYPE>)?(,sourceKey=<SOURCE_KEY>)?(,sourceValue=<SOURCE_VALUE>)?)"
+----
+where:
+[horizontal]
+`<NAME>`:: Specifies the name under which the binding value is going to be exposed. It can be excluded only when the `objectType` parameter is set to `Secret` or `ConfigMap`.
```

Comment:
```suggestion
`<NAME>`:: Specifies the name under which the binding value is going to be exposed. You can exclude it only when the `objectType` parameter is set to `Secret` or `ConfigMap`.
```
Not sure if this absolutely necessary in this case, but here's one way of avoiding passive voice here.

Reply:
@bburt-rh Thanks for the suggestions!

> Edit to avoid using the future tense.
The value is going to be exposed in the future and we need to specify it clearly here. Hence I am changing this to: "Specifies the name under which the binding value is to be exposed."